rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2023-03-10"
+channel = "nightly-2023-06-19"
```

Comment:
You can use https://github.com/datafuselabs/databend/pull/11711 in which I already build image for `nightly-2023-06-09`